### PR TITLE
DED-260 reorganizing authorisation content

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/authorization/authorizing-api-calls/index.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-weight: 30
+weight: 10
 category: "authorization"
 subcategory: "authorizing-api-calls"
 title: "Authorizing API calls"
```
Comment (ale-kacperczyk) [inline src/pages/authorization/authorizing-api-calls/index.mdx:5]: Some of the links in the **Use cases** table might no longer work as the documents were moved. For example, `[Personal Access Tokens](#personal-access-tokens)` will most likely not work.

File: src/pages/authorization/authorizing-api-calls/index.mdx [hunk 2]
```diff
@@ -44,1028 +44,11 @@ You can find all the requests from the Agent and Customer authorization flows in
...
-</Section>
-
-## Revoking tokens
-
-Look at the [Customer Accounts API specification](/authorization/customer-accounts-api/#tag/Tokens/paths/~1token/delete) to find more details about revoking customer tokens.
-
-## Validating the access token
-
-The instruction on how to validate customer access token of a customer is available in the [Customer Accounts API specification](/authorization/customer-accounts-api/#tag/Tokens/paths/~1info/get).
+To find details about making calls to [Customer Chat API](/messaging/customer-chat-api/), please see the [Customer Authorisation](/authorization/customer-authorization/).
```
Comment (ale-kacperczyk) [inline src/pages/authorization/authorizing-api-calls/index.mdx:51]:
```suggestion
To find details about authorizing calls to the [Customer Chat API](/messaging/customer-chat-api/), please see [Customer Authorization](/authorization/customer-authorization/).
```

File: src/pages/authorization/authorizing-api-calls/index.mdx [hunk 3]
```diff
@@ -44,1028 +44,11 @@ You can find all the requests from the Agent and Customer authorization flows in
...
-    "organization_id": "390e44e6-f1e6-0368c-z6ddb-74g14508c2ex",
-    "scope": "chats--all:ro,chats--all:rw",
-    "token_type": "Bearer"
-}
-```
-
-</CodeResponse>
-</Code>
-</Section>
+To find details about making calls to the [Agent Chat API](/messaging/agent-chat-api/) and [Configuration API](/management/configuration-api/), please see the [Agent Authorization](/authorization/agent-authorization/).
```
Comment (ale-kacperczyk) [inline src/pages/authorization/authorizing-api-calls/index.mdx:47]:
```suggestion
To find details about authorizing calls to the [Agent Chat API](/messaging/agent-chat-api/) and the [Configuration API](/management/configuration-api/), please see [Agent Authorization](/authorization/agent-authorization/).
```

File: src/pages/authorization/agent-authorization/index.mdx [hunk 4]
```diff
@@ -0,0 +1,664 @@
+---
+weight: 20
+category: "authorization"
+subcategory: "agent-authorization"
+title: "Agent Authorization"
+tagline: "Going through authorization in practice."
+desc: "Learn how to authorize your app to send a sample request using our Messaging APIs. A step-by-step tutorial that guides you through the process."
+---
+
+# Agent authorization flows
```
Comment (ale-kacperczyk) [inline src/pages/authorization/agent-authorization/index.mdx:10]:
```suggestion
# Introduction

To call LiveChat APIs, you need to authorize your calls with an access token. This document describes three authorization flows that let you acquire a token:

- Personal Access Token
- Implicit flow
- Authorization code grant
```

File: src/pages/authorization/agent-authorization/index.mdx [hunk 5]
```diff
@@ -0,0 +1,664 @@
...
+
+All of the following authorization flows let your app acquire an `access_token`. With this token, the app can call the following APIs on behalf of an agent:
+
+- the Agent Chat API
+- the Configuration API
+- LiveChat Accounts API
+- HelpDesk API
+- Reports API
+
+To make calls to Customer Chat API, you need a different `access_token`. For that, see [Customer Authorisation](/authorization/customer-authorization/).
```
Comment (ale-kacperczyk) [inline src/pages/authorization/agent-authorization/index.mdx:20]:
```suggestion
To make calls to Customer Chat API, you need a different `access_token`. For that, see [Customer Authorisation](/authorization/customer-authorization/).

# Postman collection

<here you can copy the content from the "Authorizing API calls" document>
```

File: src/pages/authorization/agent-authorization/index.mdx [hunk 6]
```diff
@@ -0,0 +1,664 @@
...
+category: "authorization"
+subcategory: "agent-authorization"
+title: "Agent Authorization"
+tagline: "Going through authorization in practice."
+desc: "Learn how to authorize your app to send a sample request using our Messaging APIs. A step-by-step tutorial that guides you through the process."
+---
+
+# Agent authorization flows
+
+All of the following authorization flows let your app acquire an `access_token`. With this token, the app can call the following APIs on behalf of an agent:
```
Comment (ale-kacperczyk) [inline src/pages/authorization/agent-authorization/index.mdx:12]:
```suggestion
With the acquired token, you can the following APIs on behalf of an agent:
```

File: src/pages/authorization/agent-authorization/index.mdx (hunk 5 above)
Comment (ale-kacperczyk) [inline src/pages/authorization/agent-authorization/index.mdx:20]:
```suggestion
To make calls to Customer Chat API, you need a different `access_token`. For that, see [Customer Authorization](/authorization/customer-authorization/).
```

File: src/pages/authorization/agent-authorization/index.mdx (hunk 4 above)
Comment (ale-kacperczyk) [reply src/pages/authorization/agent-authorization/index.mdx:10]: The same changes would apply to the Customer authorization doc

File: src/pages/authorization/agent-authorization/index.mdx [hunk 7]
```diff
@@ -0,0 +1,676 @@
...
+    "expires_in": 28725,
+    "organization_id": "390e44e6-f1e6-0368c-z6ddb-74g14508c2ex",
+    "scope": "chats--all:ro,chats--all:rw",
+    "token_type": "Bearer"
+}
+```
+
+</CodeResponse>
+</Code>
+</Section>
```
Comment (ale-kacperczyk) [inline src/pages/authorization/agent-authorization/index.mdx:676]: # Errors
`<link to /authorization/authorizing-api-calls/#errors>`

File: src/pages/authorization/customer-authorization/index.mdx [hunk 8]
```diff
@@ -0,0 +1,383 @@
...
+</Code>
+</Section>
+
+# Revoking tokens
+
+Look at the [Customer Accounts API specification](/authorization/customer-accounts-api/#tag/Tokens/paths/~1token/delete) to find more details about revoking customer tokens.
+
+# Validating the access token
+
+The instruction on how to validate customer access token of a customer is available in the [Customer Accounts API specification](/authorization/customer-accounts-api/#tag/Tokens/paths/~1info/get).
```
Comment (ale-kacperczyk) [inline src/pages/authorization/customer-authorization/index.mdx:383]: # Errors
`<link to /authorization/authorizing-api-calls#errors>`

File: src/pages/authorization/agent-authorization/index.mdx (hunk 7 above)
Comment (ale-kacperczyk) [reply src/pages/authorization/agent-authorization/index.mdx:676]: We left the Errors section in the Authorizing API calls, which is not optimal, as errors should be close to methods. But we would have to duplicate the content - paste it into the Agent authorization and Customer authorization docs, which isn't optimal either :/ 

I propose to add the Errors section in both Agent authorization and Customer authorization and link to Erros in the Authorizing API calls document.

File: src/pages/authorization/customer-authorization/index.mdx [hunk 9]
```diff
@@ -0,0 +1,383 @@
...
+
+```
+Set-Cookie: __lc_cid=3aa138c1-c137-41c6-6b26-cface5857378; Path=/customer; Domain=accounts.livechat.com; Expires=Sun, 18 Jun 2023 12:26:02 GMT; Max-Age=63072000; HttpOnly; Secure; SameSite=None
+
+Set-Cookie: __lc_cst=87b9d0222e63e349da85535d07c81e1e1fb938de41f4e16e1d9ba5965f25db4aeea46b39d0fb; Path=/customer; Domain=accounts.livechat.com; Expires=Sun, 18 Jun 2023 12:26:02 GMT; Max-Age=63072000; HttpOnly; Secure; SameSite=None
+```
+
+Your web browser will automatically store and attach these cookies to future requests when you try to acquire a new access token.
+
+💡 **Got stuck? See common** [**Errors**](#errors).
```
Comment (ale-kacperczyk) [inline src/pages/authorization/customer-authorization/index.mdx:73]:
```suggestion
💡 **Got stuck? See common** [**Errors**](/authorization/authorizing-api-calls#errors).
```

Comment (ale-kacperczyk) [reply src/pages/authorization/customer-authorization/index.mdx:73]: There are more such links

File: src/pages/authorization/agent-authorization/index.mdx [hunk 10]
```diff
@@ -0,0 +1,676 @@
...
+
+💡 To see code samples with PKCE parameters, use the dropdown list.
+
+<div style={{maxWidth: '400px'}}>
+  <img src="/images/authorization/livechat-code-samples-versions-2x-small.png" alt="LiveChat code samples with PKCE" width="400px" height="185px"/>
+</div>
+
+## Step 1: Create an app
+
+If you don't have an app yet, start by creating one in <a href="https://developers.livechat.com/console/" target="_blank">**Developer Console**</a>. The app is a LiveChat OAuth 2.1 Client with its **Id** and **Secret**. You need to configure the Authorization building block of your application. [See the guide](/getting-started/authorization/authorization-in-practice/#step-1-configure-the-authorization-building-block) to learn how to do that.
```
Comment (ale-kacperczyk) [inline src/pages/authorization/agent-authorization/index.mdx:285]: **See the guide** doesn't work

File: src/pages/authorization/faq/index.mdx [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ---
-weight: 70
+weight: 60
 category: "authorization"
 subcategory: "faq"
 title: "Authorization FAQ"
```
Comment (ale-kacperczyk) [inline src/pages/authorization/faq/index.mdx:5]: You probably need to check links in this document as well

File: src/pages/authorization/agent-authorization/index.mdx [hunk 12]
```diff
@@ -0,0 +1,676 @@
+---
+weight: 20
+category: "authorization"
+subcategory: "agent-authorization"
+title: "Agent Authorization"
+tagline: "Going through authorization in practice."
+desc: "Learn how to authorize your app to send a sample request using our Messaging APIs. A step-by-step tutorial that guides you through the process."
+---
```
Comment (ale-kacperczyk) [inline src/pages/authorization/agent-authorization/index.mdx:8]: You need to go through our other API documents (Agent Chat, Customer, COnfiguration, Reports, etc.) and check the Authorization section. There are links that might need to be updates, for example https://developers.livechat.com/docs/messaging/agent-chat-api#authorization

File: src/pages/authorization/agent-authorization/index.mdx (hunk 7 above)
Comment (kamilatokarska) [reply src/pages/authorization/agent-authorization/index.mdx:676]: Okay, I agree that seems most reasonable

File: src/pages/authorization/agent-authorization/index.mdx (hunk 12 above)
Comment (kamilatokarska) [reply src/pages/authorization/agent-authorization/index.mdx:8]: Okay, I did not think about that. I am doing it now

File: src/pages/messaging/agent-chat-api/index.mdx [hunk 13]
```diff
@@ -34,11 +34,11 @@ The basics on authorization, access tokens, and scopes in the Agent Chat Web API
 
 ### Authorization
 
-**Agent authorization** is handled with access tokens. Find out how to get an **access token** from [Agent authorization flows](/authorization/authorizing-api-calls/#agent-authorization-flows). Take note **you won't be able to use this token** for the Customer Chat API.
+**Agent authorization** is handled with access tokens. Find out how to get an **access token** from [Agent authorization flows](/authorization/agent-authorization). Take note **you won't be able to use this token** for the Customer Chat API.
 
 ### Scopes
 
-All authorization scopes are listed in the [Scopes](/getting-started/authorization/scopes/) document. If a method requires certain scopes, you'll find them included in the method description. Keep in mind that Web API requires authorization every time you make a request.
+All authorization scopes are listed in the [Scopes](/authorization/scopes) document. If a method requires certain scopes, you'll find them included in the method description. Keep in mind that Web API requires authorization every time you make a request.
```
Comment (ale-kacperczyk) [inline src/pages/messaging/agent-chat-api/index.mdx:41]:
```suggestion
All authorization scopes are listed in the [Scopes](/authorization/scopes/) document. If a method requires certain scopes, you'll find them included in the method description. Keep in mind that Web API requires authorization every time you make a request.
```

File: src/pages/authorization/faq/index.mdx [hunk 14]
```diff
@@ -11,12 +11,12 @@ desc: "Common problems and frequently asked questions about LiveChat authorizati
...
 
-For fast & easy authorization, we recommend [**Personal Access Tokens (PATs)**](/authorization/authorizing-api-calls/#personal-access-tokens). They're a way to go if you want to authorize private apps or speed up the development process.
+For fast & easy authorization, we recommend [**Personal Access Tokens (PATs)**](/authorization/agent-authorization#personal-access-tokens). They're a way to go if you want to authorize private apps or speed up the development process.
 
 #### Can I use the same access token to make calls to all APIs?
 
-No. Making calls to the Customer Chat API requires you to use a different token than for all other APIs. Read about [**Agent authorization flows**](/authorization/authorizing-api-calls/#agent-authorization-flows)
-and [**Customer authorization flows**](/authorization/authorizing-api-calls/#customer-authorization-flow).
+No. Making calls to the Customer Chat API requires you to use a different token than for all other APIs. Read about [**Agent authorization flows**](/authorization/agent-authorization)
+and [**Customer authorization flows**](/authorization/customer-authorization).
```
Comment (ale-kacperczyk) [inline src/pages/authorization/faq/index.mdx:19]:
```suggestion
and [**Customer authorization flows**](/authorization/customer-authorization/).
```

File: src/pages/authorization/faq/index.mdx [hunk 15]
```diff
@@ -11,12 +11,12 @@ desc: "Common problems and frequently asked questions about LiveChat authorizati
...
 #### What's the easiest way to authorize calls to the LiveChat API?
 
-For fast & easy authorization, we recommend [**Personal Access Tokens (PATs)**](/authorization/authorizing-api-calls/#personal-access-tokens). They're a way to go if you want to authorize private apps or speed up the development process.
+For fast & easy authorization, we recommend [**Personal Access Tokens (PATs)**](/authorization/agent-authorization#personal-access-tokens). They're a way to go if you want to authorize private apps or speed up the development process.
 
 #### Can I use the same access token to make calls to all APIs?
 
-No. Making calls to the Customer Chat API requires you to use a different token than for all other APIs. Read about [**Agent authorization flows**](/authorization/authorizing-api-calls/#agent-authorization-flows)
-and [**Customer authorization flows**](/authorization/authorizing-api-calls/#customer-authorization-flow).
+No. Making calls to the Customer Chat API requires you to use a different token than for all other APIs. Read about [**Agent authorization flows**](/authorization/agent-authorization)
```
Comment (ale-kacperczyk) [inline src/pages/authorization/faq/index.mdx:18]:
```suggestion
No. Making calls to the Customer Chat API requires you to use a different token than for all other APIs. Read about [**Agent authorization flows**](/authorization/agent-authorization/)
```

File: src/pages/authorization/faq/index.mdx [hunk 16]
```diff
@@ -54,16 +54,16 @@ curl 'https://accounts.livechat.com/customer/' \
...
 
 ## Errors
 
 #### I get the "Invalid access token" error.
 
 1. Perhaps, you're trying to authorize your call to the **Customer Chat API** with an access token valid for the **Agent Chat API** & **Configuration API**, or the other way round.
 
-   - [**Get a token for Agent Chat and Configuration APIs**](/authorization/authorizing-api-calls/#agent-authorization-flows)
-   - [**Get a token for Customer Chat API**](/authorization/authorizing-api-calls/#customer-authorization-flow)
+   - [**Get a token for Agent Chat and Configuration APIs**](/authorization/agent-authorization)
```
Comment (ale-kacperczyk) [inline src/pages/authorization/faq/index.mdx:65]:
```suggestion
   - [**Get a token for Agent Chat and Configuration APIs**](/authorization/agent-authorization/)
```

File: src/pages/authorization/faq/index.mdx [hunk 17]
```diff
@@ -54,16 +54,16 @@ curl 'https://accounts.livechat.com/customer/' \
...
+   - [**Get a token for Customer Chat API**](/authorization/customer-authorization)
```
Comment (ale-kacperczyk) [inline src/pages/authorization/faq/index.mdx:66]:
```suggestion
   - [**Get a token for Customer Chat API**](/authorization/customer-authorization/)
```

File: src/pages/management/configuration-api/index.mdx [hunk 18]
```diff
@@ -20,7 +20,7 @@ This document describes the **Configuration API v3.4**. This is the latest stabl
 
 ## Authentication
 
-Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/authorizing-api-calls/#agent-authorization-flows). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
+Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/agent-authorization). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/index.mdx:23]:
```suggestion
Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/agent-authorization/). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
```

File: src/pages/management/configuration-api/v3.2/index.mdx [hunk 19]
```diff
@@ -22,7 +22,7 @@ API responses will contain one of two headers related to the API lifecycle: `Leg
 
 ## Authentication
 
-Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/authorizing-api-calls/#agent-authorization-flows). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
+Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/agent-authorization). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/v3.2/index.mdx:25]:
```suggestion
Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/agent-authorization/). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
```

File: src/pages/management/configuration-api/v3.3/index.mdx [hunk 20]
```diff
@@ -22,7 +22,7 @@ API responses will contain one of two headers related to the API lifecycle: `Leg
 
 ## Authentication
 
-Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/authorizing-api-calls/#agent-authorization-flows). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
+Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/agent-authorization). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/v3.3/index.mdx:25]:
```suggestion
Authentication for Configuration API is handled by access tokens. Find out how to get an access token from [Agent authorization flows](/authorization/agent-authorization/). If a method requires particular authorization scopes, you’ll find them included in the method description. Each request should contain the `Authorization: Bearer <your_access_token>` header.
```

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**